pkgs/by-name/re/renode-bin/package.nix

```diff
@@ -0,0 +1,108 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  fetchurl,
+  fetchpatch,
+  autoPatchelfHook,
+  makeWrapper,
+  nix-update-script,
+  glibcLocales,
+  python3Packages,
+  dotnetCorePackages,
+  gtk-sharp-3_0,
+  gtk3-x11,
+  dconf,
+}:
+
+let
+  pythonLibs =
+    with python3Packages;
+    makePythonPath [
+      construct
+      psutil
+      pyyaml
+      requests
+      tkinter
+
+      # from tools/csv2resd/requirements.txt
+      construct
+
+      # from tools/execution_tracer/requirements.txt
+      pyelftools
+
+      (robotframework.overrideDerivation (oldAttrs: {
+        src = fetchFromGitHub {
+          owner = "robotframework";
+          repo = "robotframework";
+          rev = "v6.1";
+          hash = "sha256-l1VupBKi52UWqJMisT2CVnXph3fGxB63mBVvYdM1NWE=";
+        };
+        patches = (oldAttrs.patches or [ ]) ++ [
+          (fetchpatch {
+            # utest: Improve filtering of output sugar for Python 3.13+
+            name = "python3.13-support.patch";
+            url = "https://github.com/robotframework/robotframework/commit/921e352556dc8538b72de1e693e2a244d420a26d.patch";
+            hash = "sha256-aSaror26x4kVkLVetPEbrJG4H1zstHsNWqmwqOys3zo=";
+          })
+        ];
+      }))
```

Comment:
I think we will probably need to create a separate delegation where we can include both in the source and binary builds.